src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2523,9 +2523,42 @@ export class PostgresStorageAdapter implements StorageAdapter {
     return result;
   }
 
-  // TODO: implement?
-  ensureIndex(): Promise<void> {
-    return Promise.resolve();
+  ensureIndex(
+    className: string,
+    schema: SchemaType,
+    fieldNames: string[],
+    indexName: ?string,
+    caseInsensitive: boolean = false
+  ): Promise<void> {
+    const defaultIndexName = `parse_default_${fieldNames.sort().join('_')}`;
```

Comment:
Can you add tests cases for this?

Reply:
Done, added a test case that uses the default name